www/docs/reference/env-vars.mdx

```diff
@@ -3,22 +3,148 @@ title: Environment Variables
+| OUTBOUND_EMAIL_FROM         | From address to use for sending outbound emails. If not set, sending email will be disabled (default).       | string                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | generated from other env vars  |
+| OUTBOUND_EMAIL_ADAPTER      | Method to use for sending outbound email.                                                                    | One of `Swoosh.Adapters.AmazonSES`, `Swoosh.Adapters.CustomerIO`, `Swoosh.Adapters.Dyn`, `Swoosh.Adapters.ExAwsAmazonSES`, `Swoosh.Adapters.Gmail`, `Swoosh.Adapters.MailPace`, `Swoosh.Adapters.Mailgun`, `Swoosh.Adapters.Mailjet`, `Swoosh.Adapters.Mandrill`, `Swoosh.Adapters.Postmark`, `Swoosh.Adapters.ProtonBridge`, `Swoosh.Adapters.SMTP`, `Swoosh.Adapters.SMTP2GO`, `Swoosh.Adapters.Sendgrid`, `Swoosh.Adapters.Sendinblue`, `Swoosh.Adapters.Sendmail`, `Swoosh.Adapters.SocketLabs`, `Swoosh.Adapters.SparkPost`, `FzHttpWeb.Mailer.NoopAdapter` | `FzHttpWeb.Mailer.NoopAdapter` |
+| OUTBOUND_EMAIL_ADAPTER_OPTS | Adapter configuration, for list of options see [Swoosh Adapters](https://github.com/swoosh/swoosh#adapters). | JSON-encoded map                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | `{}`                           |
+
+### Connectivity Checks
+
+| Env Key                      | Description                                                                                                                    | Format  | Default |
+| ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------ | ------- | ------- |
+| CONNECTIVITY_CHECKS_ENABLED  | Enable / disable periodic checking for egress connectivity. Determines the instance's public IP to populate `Endpoint` fields. | boolean | true    |
+| CONNECTIVITY_CHECKS_INTERVAL | Periodicity in seconds to check for egress connectivity.                                                                       | integer | 3600    |
```

Comment:
Ah I think I had updated this to be `43200` by default -- twice a day (trying to reduce amount of ingested data).
```suggestion
| CONNECTIVITY_CHECKS_INTERVAL | Periodicity in seconds to check for egress connectivity.                                                                       | integer | 43200    |
```